spec-main/api-app-spec.ts

```diff
@@ -140,7 +119,7 @@ describe('app module', () => {
 
   describe('app.isPackaged', () => {
     it('should be false durings tests', () => {
-      expect(app.isPackaged).to.be.false()
+      expect(app.isPackaged).equal(false)
```

Comment:
```suggestion
      expect(app.isPackaged).to.equal(false)
```

also why not `to.be.false`?

Reply:
@nornagon Properties are super bad for assertions.  I.e. `to.be.abc` is a valid property.  `dirty-chai` isn't typed correctly either so that doesn't work well in this case